README.md

```diff
@@ -59,10 +59,20 @@ sudo yum install git valgrind autoconf automake flex bison python-docutils pytho
 
 #### Ubuntu
 
-Note: this list is (obviously) incomplete. We intend to complete it as we 
-have more information and install additional systems.
+Add Adiscon repository:
```

Comment:
It would be great if we could also add that the repro ist not necessarily needed. You can also build everything from scratch (this is useful to do for developers who use master branch, as it may require things newer than even in the v8-devel repo!).